stories/button/__snapshots__/button.stories.storyshot

```diff
@@ -22,23 +22,23 @@ exports[`Storyshots Components/Button Button styles 1`] = `
    
         
   <button
-    class="fd-button"
+    class="fd-button fd-button--emphasized"
   >
-    Default Button
```

Comment:
What about these styles?

Default button: is used for neutral or informative actions (secondary action on the screen).
Emphasized button: is used to indicate the primary action on the screen. 
Ghost button: is used to trigger secondary actions However, if a page also requires an important section highlighted in a content toolbar, then use the ghost style.
Attention button: is used to trigger a semantic action that needs attention and should always be accompanied by text. 
Transparent button: is used to trigger a negative path action within a header or footer bar, and secondary actions in toolbars.